cms/djangoapps/coursegraph/management/commands/dump_to_neo4j.py

```diff
@@ -1,6 +1,17 @@
 """
 This file contains a management command for exporting the modulestore to
-neo4j, a graph database.
+Neo4j, a graph database.
+
+Example usages:
```

Comment:
Note: Moved the example usage up to the module docstring because they don't display well in command-line helptext. Helptext is all displayed in a single line.